fre/pp/checkoutScript.py

```diff
@@ -34,14 +34,31 @@ def _checkoutTemplate(experiment, platform, target):
     # Set the name of the directory
     name = f"{experiment}__{platform}__{target}"
 
-    # Get name of local branch and check branch rquest
-    branch_names = subprocess.run(["git","branch"],capture_output=True, text=True)
-    local_branch_name = branch_names.stdout.split()[1]
-    if branch != 'main':
-        if branch != local_branch_name:
-            stop_report = ("Error in branch: local branch does not match branch input")
-            sys.exit(stop_report)
-            return 1
+    # branch and version parameters
+    default_tag = subprocess.run(["fre","--version"],capture_output=True, text=True).stdout.split()[2]
```

Comment:
The tag retrieved from `fre --version` has a one digit month but the fre-workflows (and fre-cli) tags are two-digit month. So this

c2b:~%>fre --version 
fre-cli | 2024.1

needs to be expanded to `2024.01` to be used as a tag for fre-workflows.

Reply:
I "fixed" this in https://github.com/NOAA-GFDL/fre-cli/pull/249/commits/822900b84049bb997c39812576b1fcd1f184c628 but it's pretty ugly so I think we can remove this if now.